packages/server/src/queries/complex/assets/index.ts

```diff
@@ -17,6 +17,7 @@ export type Asset = {
   coinImageUrl?: string;
   isVerified: boolean;
   isUnstable: boolean;
+  sourceDenom: string;
```

Comment:
I'm thinking we could keep this type simple and include sourceDenom in an asset type defined in bridge.ts in this folder, to be used by a bridge tRPC router. It would make more sense in that context IMO. Somewhat of a nit IMO

Reply:
I agree that we should keep this function as simple as possible. However, since `sourceDenom` is a crucial part of the asset, it seems odd that it's not included in the `getAsset` method. Otherwise, we would have to repeatedly find the asset in the asset list to get the `sourceDenom`